### PR TITLE
Topic2

### PR DESCRIPTION
**Pull request by fcuenca4**
Final commit without extras.
i have deleted the "target files" in /ext/.

### CODE REVIEW

File: ex3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <groupId>org.fcuenca</groupId>
+  <artifactId>ex3</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>ex3</name>
+  <modules>
+  	<module>../ex1</module>
+  	<module>../ex2</module>
+  </modules>
+</project>
```
Comment (hernan-globant) [inline ex3/pom.xml:13]: The module part shouldn't be part of exercise. The exercise #3 stays to start adding some dependencies (libraries), adding some source code, and customizing the project to suit specific needs. 
Also, for the multi-module project exercise, it's needed to update the first pom you defined on exercise #1

File: ex1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+</project>
```
Comment (hernan-globant) [inline ex1/pom.xml:25]: Correct, good!

File: ex2/pom.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+  <build>
+    <finalName>ex2</finalName>
+  </build>
+</project>
```
Comment (hernan-globant) [inline ex2/pom.xml:21]: Did you want to create a pom for the web application, stated on exercise #4?

File: ex2/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE web-app PUBLIC
+ "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN"
+ "http://java.sun.com/dtd/web-app_2_3.dtd" >
+
+<web-app>
+  <display-name>Archetype Created Web Application</display-name>
+</web-app>
```
Comment (hernan-globant) [inline ex2/src/main/webapp/WEB-INF/web.xml:7]: This file isn't needed to be uploaded.

**Comment by lucast-globant** [comment]
Well Done! Continue with next topics!

**Pull request closed without merging**